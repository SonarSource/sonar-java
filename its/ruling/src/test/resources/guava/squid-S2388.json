{
'com.google.guava:guava:src/com/google/common/cache/LocalCache.java':[
1295,
1618,
1665,
],
'com.google.guava:guava:src/com/google/common/collect/AbstractMapBasedMultimap.java':[
611,
773,
926,
954,
969,
977,
],
'com.google.guava:guava:src/com/google/common/collect/ConcurrentHashMultiset.java':[
551,
],
'com.google.guava:guava:src/com/google/common/collect/MapMakerInternalMap.java':[
1180,
1437,
1702,
1718,
1741,
1757,
],
}
