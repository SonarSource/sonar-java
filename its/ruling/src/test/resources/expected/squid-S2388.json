{
'project:com/sun/jmx/mbeanserver/WeakIdentityHashMap.java':[
124,
],
'project:java/util/ArrayList.java':[
1005,
],
'project:java/util/TreeMap.java':[
1394,
1398,
1499,
1503,
1507,
1511,
],
'project:java/util/WeakHashMap.java':[
699,
],
'project:java/util/concurrent/CopyOnWriteArrayList.java':[
1218,
],
'project:javax/swing/SwingWorker.java':[
842,
],
'project:javax/swing/text/ComponentView.java':[
411,
424,
446,
492,
],
}
